### PR TITLE
A bit more snippets for gRPC

### PR DESCRIPTION
**Pull request by ennru**
Trying to make the page more complete, but without proper step-by-step sources, this can't really succeed.

### CODE REVIEW

File: docs-source/docs/modules/cqrs/pages/tutorial/grpc-server.adoc [hunk 1]
```diff
@@ -84,6 +100,4 @@ Note the logging from the `ShoppingCartServiceImpl` in the console of the servic
 
 You can stop the service with `ctrl-c`.
 
-== Deploy
-
-TODO deploy to cloud and try it
+ifdef::todo:[== Deploy]
```
Comment (patriknw) [inline docs-source/docs/modules/cqrs/pages/tutorial/grpc-server.adoc:103]: I think it's better to show what we intend to include, when we ask for feedback from others

Comment (ennru) [reply docs-source/docs/modules/cqrs/pages/tutorial/grpc-server.adoc:103]: The TODO's show when using `make html-author-mode`.

Comment (patriknw) [reply docs-source/docs/modules/cqrs/pages/tutorial/grpc-server.adoc:103]: yes, but is that what we will publish (when we publish the first versions)?

Comment (ennru) [reply docs-source/docs/modules/cqrs/pages/tutorial/grpc-server.adoc:103]: If we want to, yes.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**